arca/backend/docker.py

```diff
@@ -58,8 +58,10 @@ class DockerBackend(BaseBackend):
 
     INSTALL_DEPENDENCIES = """
         FROM {name}:{tag}
-        RUN apk update
-        RUN apk add --no-cache {dependencies}
+        USER root
+        RUN apt-get update
+        RUN apt-get install --no-install-recommends -y --autoremove {dependencies}
```

Comment:
you might want to update and install and clean metadata in a oneliner not to make the layer bigger than necessary.